### DISCUSSION TITLE
Fully delete crates

### DISCUSSION THREAD

**Discussion by tembbo** [rust-lang/crates.io#10273] [General]
Why is there no option to delete a crate on crates.io. I know that you can yank them, and I know that it is too prevent incidents like the one with left-pad. But if no package depends on it, there should be no reason to not have the option to fully delete it.

**Comment by eth3lbert** [comment]
Crate deletion is now experimental and available in hidden route. You should be able to try it out using the following URL https://crates.io/crates/:name/delete.

Please see https://github.com/rust-lang/rfcs/pull/3660 for details on crate deletion. You can track the progress through https://github.com/rust-lang/crates.io/issues/9352.

**Reply by tembbo** [reply]
I get the error saying that i can't delete it because it has more than 100 downloads per month. Will i be able to delete when it's less than that again

**Reply by eth3lbert** [reply]
> I get the error saying that i can't delete it because it has more than 100 downloads per month. Will i be able to delete when it's less than that again

Thanks for trying this out!

We discussed the download limit in last crates.io team meeting and agreed to raise it. https://github.com/rust-lang/crates.io/pull/10266 is the relevant PR that is currently in progress.
